app/utilities/metadata_parser_v2.py

```diff
@@ -20,8 +20,7 @@
 
 logger = get_logger()
 
-
-VALIDATORS = {
+VALIDATORS: Mapping[str, Any] = {
```

Comment:
This should be `Callable` instead of `Any`.

Reply:
Sorted 👍